README.md

```diff
@@ -157,7 +157,7 @@ first = "Tom" last = "Preston-Werner" # INVALID
 Keys
 ----
 
-A key may be either bare, quoted or dotted.
+A key may be either bare, quoted, or dotted.
```

Comment:
Hmmm... I'm gonna let this slide through.

Reply:
I checked before making that change, the document appears to favor Oxford commas. I don't usually go off randomly changing between styles. (My goal is to get things merged, not cause style fights.)